tests/test_aggregator.py

```diff
@@ -0,0 +1,207 @@
+from mock import patch, MagicMock
+
+from django.conf import settings
+from django.test import TestCase
+
+from algoliasearch_django import Aggregator
+from algoliasearch_django import algolia_engine
+from algoliasearch_django.models import AlgoliaIndexError
+
+from .models import Website, Example, User
+from .factories import WebsiteFactory, ExampleFactory, UserFactory
+
+
+class AggregatorTestCase(TestCase):
+
+    def setUp(self):
+        self.client = MagicMock()
```

Comment:
I think this creates some room for confusion, as you're overriding default Django's TestCase client. Its not like this client will ever be used in these tests, but also, there is no particular need to override it.

Reply:
Fair point. I was following the pattern used in `test_index`, which came from upstream. This is a new test class though, so I've renamed.